src/core/util/markers.ts

```diff
@@ -40,7 +40,12 @@ export const enum AttributeMarker {
   /**
    * Component Styles.
    */
-  ComponentStyles = 'q:style',
+  ComponentStyles = 'q:sstyle',
```

Comment:
Ok guess it's correct, but why the change from `q:style` to `q:sstyle`?

Reply:
So we can have both scoped and unscoped styles.